newm-server/src/main/kotlin/io/newm/server/database/migration/V70__MarketplaceUpdates.kt

```diff
@@ -0,0 +1,22 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V70__MarketplaceUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            execInBatch(
+                listOf(
+                    "DELETE FROM marketplace_pending_orders",
```

Comment:
How is this different from V66?  Won't this file simply do nothing because the records already exist?

Reply:
Actually, these are new. The change in V66 was just to get rid of the dependency on the constants defined at ConfigRepository. I realized that doing that is bad because when we delete rows from Config table we could have to keep unused constants.